content/applications/marketing/social_marketing/essentials/social_essentials.rst

```diff
@@ -2,233 +2,415 @@
+In order to create social posts and analyze content with Odoo *Social Marketing*, social media
+accounts **must** be added as a *stream* on the application's main dashboard.
+
+.. note::
+   Be aware that personal profiles **cannot** be added as a stream. The main use of Odoo *Social
+   Marketing* is to manage and analyze business accounts on social media platforms.
+
+.. warning::
+   Odoo *Social Marketing* has some limitations in regards to social media accounts. For example,
+   Odoo **cannot** handle a large quantity of various pages (e.g. ~40 pages) under the same company.
+   The same limitations are present in a multi-company environment because of how the API is
+   constructed.
+
+.. warning::
+   In multi-company environments, if every company doesn't activate a page at once, it will result
+   in a permission error.
+
+   For example, if Company 1 is the only company selected from the main Odoo dashboard, and
+   activates *Facebook Page 1* and *Facebook Page 2*, then those pages will be accesible on the
+   *Social Marketing* dashboard.
+
+   However, if on that same database, the user adds Company 2 from the company drop-down menu in the
+   header, and attempts to add those same streams, it results in a permission error.
+
+   .. image:: social_essentials/permission-error.png
+      :align: center
+      :alt: View of the permission error that appears when incorrectly attempting to add stream.
+
+Social media streams
+--------------------
+
+To add a social media business account as a stream, navigate to :menuselection:`Social Marketing
+app` and select the :guilabel:`Add A Stream` button located in the upper-left corner. Doing so
+reveals an :guilabel:`Add a Stream` pop-up window.
 
-Add a social media stream
--------------------------
+.. image:: social_essentials/add-stream-social-popup.png
+   :align: center
+   :alt: View of the pop-up window that appears when Add a Stream is selected in Odoo.
 
-Add a social media account as a stream by navigating to :menuselection:`Social Marketing` and then
-select the :guilabel:`Add A Stream` button located in the upper left corner
+In the :guilabel:`Add a Stream` pop-up window, choose to :guilabel:`Link a new account` for a
+business from any of the following popular social media platforms: :guilabel:`Facebook`,
+:guilabel:`Instagram`, :guilabel:`LinkedIn`, :guilabel:`Twitter`, and :guilabel:`YouTube`.
 
-When :guilabel:`Add A Stream` is clicked, the following pop-up appears, displaying the different
-social media outlets to choose from.
+After clicking the desired social media outlet from the :guilabel:`Add a Stream` pop-up window, Odoo
+navigates directly to that specific social media outlet's authorization page, where permission must
+be granted, in order for Odoo to add that particular social media account as a stream to the *Social
+Marketing* application.
 
-.. image:: social_essentials/social-add-streams.png
+.. image:: social_essentials/social-marketing-dashboard.png
```

Comment:
ideally this screenshot is a bit bigger on the farther end of our dimensional limit, closer to 933px instead of 634px. It's hard to read what's going on here on desktop.